Lib/_pyio.py

```diff
@@ -87,10 +87,6 @@ def open(file, mode="r", buffering=-1, encoding=None, errors=None,
     returned as strings, the bytes having been first decoded using a
     platform-dependent encoding or using the specified encoding if given.
 
-    'U' mode is deprecated and will raise an exception in future versions
```

Comment:
It wont let me add a comment to the line as it is too far away from the changes, but shouldn't line 74 also be removed here?